### PR TITLE
feat: add AddVotableColor events

### PR DESCRIPTION
**Pull request by Gift-Naomi**
- [ ] issue #
- [ ] follows contribution [guide](https://github.com/keep-starknet-strange/art-peace/blob/main/CONTRIBUTING.md)
- [ ] breaking change

### CODE REVIEW

File: backend/routes/indexer/route.go [hunk 1]
```diff
@@ -332,6 +334,41 @@ func StartMessageProcessor() {
 	}()
 }
 
+func AddVotableColor(hex string, dayIndex int) error {
+	dbHost := os.Getenv("DB_HOST")
```
Comment (b-j-roberts) [inline backend/routes/indexer/route.go:338]: I'd recommend looking at how other indexed events are setup. Like this one https://github.com/keep-starknet-strange/art-peace/blob/bdfef62019ede21e92dd169037aa3bb328c1e350/backend/routes/indexer/vote.go#L10

There are already modules to make interacting with postgres easier for example

You'll also need to add the event setup to the indexer directory, here : https://github.com/keep-starknet-strange/art-peace/blob/main/indexer/script.js

**Comment by b-j-roberts** [comment]
@Gift-Naomi Have you been able to implement this yet?

**Comment by Gift-Naomi** [comment]
@b-j-roberts I'm still working on it, I'll be done shortly.

**Comment by Gift-Naomi** [comment]
> @b-j-roberts I'm still working on it, I'll be done shortly.

@b-j-roberts I've indexed votable colors event. Please take a look and revert!

**Comment by b-j-roberts** [comment]
Appreciate it!

**Pull request merged**